### ISSUE TITLE
Confusing message

### ISSUE THREAD

**Issue by jonas-w** [pop-os/upgrade#251]
When running the script, this message shows:
```Current Release: 21.10
Upgrading to: 22.04
New version available: false
no release available to upgrade to
```
Why does it say "upgrading to" and directly after that "new version available: false"?

**Comment by jacobgkau** [comment]
> Why does it say "upgrading to" and directly after that "new version available: false"?

Because it's showing what version it would be upgrading to, but that version is not yet available.

**Comment by jonas-w** [comment]
Yes okay but my point still stands that it's confusing. It's nothing important but still wanted to mention it here

**Comment by jacobgkau** [comment]
That's fine, I did not close your issue, just explained why it currently works the way that it does.
